### ISSUE TITLE
kitti3 -p top doesn't work with two monitors

### ISSUE THREAD

**Issue by matclab** [LandingEllipse/kitti3#9]
I have two monitors: 
```
eDP-1 connected primary 1920x1080+0+1050 (normal left inverted right x axis y axis) 294mm x 165mm
DP-1 connected 1680x1050+120+0 (normal left inverted right x axis y axis) 433mm x 270mm
```
and I launch ` kitti3 -p top -- -c=~/.config/kitty/quake.conf`.

While it works when when I have only one monitor connected, with the two monitor setup, the terminal appears centered (almost) on the upper left edge of the current monitor, which makes it unusable.

**Comment by LandingEllipse** [comment]
Hi @matclab ,

Just had a chance to debug this and develop a fix, as described in #8 . New version 0.2.6 of kitti3 is now on PYPI. I'll close this issue for now; please take the new version for a spin and let me know if there are any problems and I'll be happy to reopen this.

Cheers,
Ariel

**Comment by matclab** [comment]
It works like a charm.
Thank you very much !
